astro.config.mjs

```diff
@@ -92,7 +107,7 @@ export default defineConfig({
                         { label: 'Data manipulation clauses', link: '/cypher/data-manipulation-clauses' },
                         { label: 'Subquery', link: '/cypher/subquery' },
                         { label: 'Macros', link: '/cypher/macro' },
-                        { label: 'Transactions', link: '/cypher/transaction' },
+                        { label: 'Transactions', link: '/cypher/trnpm ansaction' },
```

Comment:
This looks like an unexpected editing. The original one looks correct and the edited one is a typo